docs/src/main/asciidoc/podman.adoc

```diff
@@ -6,72 +6,130 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Using Podman with Quarkus
 
 https://podman.io/[Podman] is a daemonless and rootless container engine for developing, managing, and running OCI Containers on your Linux system or other OS.
-Podman can be used the same way as Docker with the `podman-docker` package.
+If you're using Podman with Quarkus, some one-off setup is needed, but once it's done, you can take advantage of all of the Quarkus features.
```

Comment:
```suggestion
If you're using Podman with Quarkus, some one-off setup is needed, but once it's done, you can take advantage of all the Quarkus features.
```

@marcwrobel removed a lot of `all of` in our code and documentation recently so I'm not sure we should add some back. Probably because of https://jakubmarian.com/is-it-all-the-or-all-of-the-in-english/ . But you're the native speaker here :).

Reply:
I now know more about English grammar than I did ten minutes ago. My reading of that link and https://www.grammar-monster.com/lessons/all_of.htm is that 'all of' is correct in this context, but the 'of' is at best three unnecessary characters, and at worst weird and colloquial. So let's get rid of it.

Reply:
[Thanks again for that, IDE](https://www.jetbrains.com/help/idea/grammar.html) ;).